ts/import-csv/StickyFooter.svelte

```diff
@@ -0,0 +1,59 @@
+<!--
+Copyright: Ankitects Pty Ltd and contributors
+License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+-->
+<script lang="ts">
+    import ButtonGroup from "../components/ButtonGroup.svelte";
+    import Col from "../components/Col.svelte";
+    import LabelButton from "../components/LabelButton.svelte";
+    import Row from "../components/Row.svelte";
+    import Shortcut from "../components/Shortcut.svelte";
+    import * as tr from "../lib/ftl";
+    import { getPlatformString } from "../lib/shortcuts";
+
+    export let path: string;
+    export let onImport: () => void;
+
+    function doImport(): void {
+        if (document.activeElement instanceof HTMLElement) {
+            document.activeElement.blur();
```

Comment:
Just curious, why is this needed?

Reply:
I kept it around because I thought it might become necessary. But I guess it's not. 🙂